_topic_map.yml

```diff
@@ -197,6 +197,8 @@ Topics:
   File: default-security-policies
 - Name: Managing network policies
   File: manage-network-policies
+- Name: Auditing listening endpoints
```

Comment:
I was going to leave a reference for changing the title of the concept module to `Audit listening endpoints`, but I see in the navigation that the adjacent concept sections (Managing, Reviewing...) follow the same style.